doc/api/globals.md

```diff
@@ -474,6 +474,18 @@ changes:
 
 A browser-compatible implementation of the [`fetch()`][] function.
 
+## Class: `File`
+
+<!-- YAML
+added:
+  - v19.2.0
+  - v18.13.0
```

Comment:
Can you add `changes:` with a `REPLACEME` version with a `pr-url` that points to this PR for making it global? (`FormData` has a similar documentation.)

Reply:
I copied this part from the Blob section, which doesn't have a changes section. https://github.com/nodejs/node/blob/main/doc/api/globals.md#class-blob